src/main/java/com/krystian/ThirdTask.java

```diff
@@ -0,0 +1,91 @@
+package com.krystian;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class ThirdTask {
+    private List<String> listOfStrings = new ArrayList<>();
+
+    public void pareAllElementsToInteger(String path) throws IOException {
+        listOfStrings = Files.readAllLines(Paths.get(path));
+        System.out.println(listOfStrings.size());
+        int counter = 0;
+        for(int x = 0; x < listOfStrings.size(); x++){
+            String [] charsOfString = listOfStrings.get(x).split("");
+            String [] firstPart = Arrays.copyOfRange(charsOfString, 0, (charsOfString.length + 1)/2);
+            String [] secondPart = Arrays.copyOfRange(charsOfString, (charsOfString.length + 1)/2, charsOfString.length);
+            counter = counter + getPointForLetter(returnCommonChar(firstPart, secondPart));
+        }
+        System.out.println(counter);
+    }
+
+
+    private String returnCommonChar(String [] first,String [] second){
+        boolean stop = false;
+        String common = "";
+        for(int x = 0; x < first.length; x++){
+            for(int y = 0; y < second.length; y++){
+                if(first[x].equals(second[y])) {
+                    common = first[x];
+                    stop = true;
+                    break;
+                }
+            }
+            if(stop){
+                break;
+            }
+        }
+        return common;
+    }
+
+    private int getPointForLetter(String w){
+        int count = 0;
+        final Map<String, Integer> alphabet = new HashMap<>();
+        addIntoHashMap(alphabet);
+        try {
+            if (w == w.toLowerCase()) {
+                count = alphabet.get(w);
+            }
+            if (w == w.toUpperCase()) {
+                count = 26 + alphabet.get(w.toLowerCase());
+            }
+        }
+        catch (NullPointerException n){
+            System.out.println("go on");
+        }
+        return count;
+    }
+
+    public void addIntoHashMap(Map<String,Integer> alphabet){
```

Comment:
zamiast tej padliny mozna zrobic 
for (char ch = 'A'; ch <= 'Z'; ++ch) 
  map.put(String.valueOf(ch), 0);  
tylko wiadomo troche by to trzeba bylo zedytowac zeby Ci licznik zwiekszalo zamiast zera